src/views/resultView.js

```diff
@@ -0,0 +1,31 @@
+import { USER_INTERFACE_ID } from '../constants.js';
+import { selectedAnswers } from '../data.js';
+
+const showResult = () => {
+  const userInterface = document.getElementById(USER_INTERFACE_ID);
```

Comment:
We should avoid accessing the DOM in the views directory, instead you should return resultElement from this function, and remove the references to userInterface. Then add a resultPage.js to the pages directory, and in there call this function and append the result to the userInterface.